websites/C/Codecademy/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "d0gg3r",
+    "id": "460318444487704597"
+  },
+  "url": ["www.codecademy.com", "news.codecademy.com"],
+  "description": {
+    "en": "Learn the technical skills you need for the job you want. As leaders in online education and learning to code, we’ve taught over 45 million people using a tested curriculum and an interactive learning environment. Start with HTML, CSS, JavaScript, SQL, Python, Data Science, and more."
+  },
+  "service": "Codecademy",
+  "version": "1.1.2",
+  "logo": "https://i.imgur.com/TnKcxoz.jpg",
+  "thumbnail": "https://i.imgur.com/5rpD3o0.png",
```

Comment:
Please change the thumbnail to a promotional image or a screenshot of the website.

Reply:
Resolved in fd75ba5.